scripts/setup/setup-reportstream.sh

```diff
@@ -16,6 +16,16 @@ echo "Resetting the database and loading the baseline settings..."
 ./gradlew reloadTables
 ./gradlew reloadSettings
 
+# Update RS Configuration for the TI API based on docker or gradle
+# PS: Logic to determine to use gradle vs docker needs updating
+# The logic needs to check if BOTH TI and RS are running in docker, otherwise it should use localhost:8080
```

Comment:
Could we put an echo or two here notifying people of the behavior in case it's not known or that someone forgets?